content/docs/connectors/keystone.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Authentication Through OpenStack Keystone"
+linkTitle: "OpenStack Keystone"
+description: ""
+date: 2020-09-30
+draft: false
+toc: true
+weight: 2100
+---
+
+## Overview
+
+[Keystone](https://docs.openstack.org/keystone/latest/) is an OpenStack service that provides API client authentication, service discovery, and distributed multi-tenant authorization. 
+
+OpenStack Keystone connector supports `offline_access` and `groups` scopes. To use this connector, you need to create a domain and user with an admin role and then specify credentials in the configuration file like in the example below.
+Keystone should expose [Identity API v3](https://docs.openstack.org/api-ref/identity/v3/) to work with dex.
```

Comment:
The term "should" is throwing me off a little here. Makes me think that sometimes Keystone doesn't expose the API... If that's the case, should we add a workaround?

```suggestion
Keystone exposes the [Identity API v3](https://docs.openstack.org/api-ref/identity/v3/) to work with dex.
```